libs/dh/shared/release-toggle/src/lib/dh-release-toggle.service.ts

```diff
@@ -0,0 +1,245 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { Injectable, computed, DestroyRef, inject } from '@angular/core';
+import { query } from '@energinet-datahub/dh/shared/util-apollo';
+import { interval, tap } from 'rxjs';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+import { GetReleaseTogglesDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DhReleaseToggleService {
+  private static readonly POLLING_INTERVAL_MS = 60_000; // Poll every minute
+  private static readonly MAX_CONSECUTIVE_RETRIES = 10; // Stop after 10 consecutive failures
+
+  // Angular-initialized properties
+  private readonly destroyRef = inject(DestroyRef);
+
+  // Service configuration
+  private readonly pollingInterval = DhReleaseToggleService.POLLING_INTERVAL_MS;
+  private readonly maxRetries = DhReleaseToggleService.MAX_CONSECUTIVE_RETRIES;
+
+  // Internal state
+  private failureCount = 0;
+
+  // Apollo query setup
+  private readonly togglesQuery = query(GetReleaseTogglesDocument, {
+    fetchPolicy: 'network-only',
+    errorPolicy: 'all', // Continue even if there are errors
+  });
+
+  // Computed signal that maps toggle data to a record for easy lookup
+  private readonly togglesMap = computed(() => {
+    const data = this.togglesQuery.data();
+    if (!data?.releaseToggles) {
+      return {};
+    }
+
+    // Assume releaseToggles is an array with objects that have 'name' and 'enabled' properties
```

Comment:
nitpick: sounds like an AI comment, the releaseToggles is typed and is indeed an array with objects that have 'name' and 'enabled' properties. i would just remove it